### ISSUE TITLE
Useless assignment found by static analyzer?

### ISSUE THREAD

**Issue by 360CodeSafe** [cjlin1/libsvm#121]
Hi all,
This is Qihoo360 CodeSafe Team, we found a useless assignment in `libsvm`, see
https://github.com/cjlin1/libsvm/blob/88a1881f03ca139beff93170d7e6f36477fabe54/svm.cpp#L2932. The return value of `strtok(line, ":")` assigned to `p`, and this value is never read!

This is not a fatal issue, and I can't provide a perfect fix for that! 

Could you help me confirm this, @cjlin1!

Thanks in advance!

**Comment by cjlin1** [comment]
Yes, for that assignment the returned value is not used. But I guess it's ok to leave that there.

360 CodeSafe writes:

**Comment by 360CodeSafe** [comment]
@cjlin1 Hmm, you are right, it is not an issue that worth the effort to fix for now.
